src/component/project/ProjectEnvironment/ProjectEnvironment.tsx

```diff
@@ -0,0 +1,141 @@
+import { useContext, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import ConditionallyRender from '../../common/ConditionallyRender';
+import { useStyles } from './ProjectEnvironment.styles';
+
+import useLoading from '../../../hooks/useLoading';
+import PageContent from '../../common/PageContent';
+import AccessContext from '../../../contexts/AccessContext';
+import HeaderTitle from '../../common/HeaderTitle';
+import { UPDATE_PROJECT } from '../../AccessProvider/permissions';
+
+import ApiError from '../../common/ApiError/ApiError';
+import useToast from '../../../hooks/useToast';
+import useUiConfig from '../../../hooks/api/getters/useUiConfig/useUiConfig';
+import useEnvironments from '../../../hooks/api/getters/useEnvironments/useEnvironments';
+import useProject from '../../../hooks/api/getters/useProject/useProject';
+import { FormControlLabel, FormGroup, Switch } from '@material-ui/core';
+import useProjectApi from '../../../hooks/api/actions/useProjectApi/useProjectApi';
+import EnvironmentDisableConfirm from './EnvironmentDisableConfirm/EnvironmentDisableConfirm';
+
+export interface ProjectEnvironment {
+    name: string;
+    enabled: boolean;
+}
+
+const ProjectListNew = () => {
```

Comment:
Perhaps we could calls this ProjectEnvironmentList? We already have a ProjectList that serves a different purpose.